apps/cms.c

```diff
@@ -1333,6 +1338,39 @@ int cms_main(int argc, char **argv)
                 ret = verify_err + 32;
             goto end;
         }
+        if (flags & CMS_VERIFY_PARTIAL) {
```

Comment:
```suggestion
        if ((flags & CMS_VERIFY_PARTIAL) != 0) {
```
according to coding style requirements

Reply:
Maybe do the new signer info printing only in case `sk_CMS_SignerInfo_num(sinfos) > 1`,
since otherwise it is implicitly clear from the verification result.